Lib/gftools/scripts/font_dependencies.py

```diff
@@ -0,0 +1,67 @@
+"""
+gftools font-dependencies
+
+Add the dependencies used to generate the font to the font's Debg table or
+dump a font's dependencies in order to make a pip requirements file.
+
+Usage:
+
+Build font dependencies:
+gftools font-dependencies build font.ttf -o font-deps.ttf
```

Comment:
(bikeshedding) I think "store" reads better than "build" to describe the action this sub-command does. It's not really building anything, just retrieving and storing some piece of data in the font.
Or maybe 'read' and 'write' would also be appropriate.